### ISSUE TITLE
Title not updated accross sites

### ISSUE THREAD

**Issue by jaydensmith** [verbb/navigation#230]
**Description**
When updating a node's title, you need to update it on every site even with "Propagate nodes across all enabled sites?" turned on.

**Comment by engram-design** [comment]
This is correct behaviour, with how Craft handles content propagation at least. To illustrate, this same behaviour happens with entries. 

Creating a section on a multi-site install, enabled for multiple sites. Create an entry with title "Test". See the same entry on all other sites. Go back to your primary site, and edit the title to "Testing". Notice all other site entries still show "Test".

I'm up for changing this behaviour if it's confusing, but I also want to stay in line with Craft's content propagation mechanisms.

**Comment by jaydensmith** [comment]
Hmm, interesting. It would be nice to be able to have an option - I have a multisite with 6 sites, it takes a while to rename 6+ nodes.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.1](https://github.com/verbb/navigation/releases/tag/4.0.0-beta.1). 

Be sure to check out the updated [docs](https://verbb.io/craft-plugins/navigation/docs/v4/get-started/installation-setup).
